networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/topics/validation/AttestationValidator.java

```diff
@@ -41,6 +41,7 @@
 
 public class AttestationValidator {
 
+  private static final UnsignedLong MAX_FUTURE_SLOT_ALLOWANCE = UnsignedLong.valueOf(10);
```

Comment:
10 is probably too generous actually.  Really the only reason for this to happen is clock drift or if Teku is lagging behind slightly so 99% of the time it will only be one slot ahead so I'd probably limit it to something like 3 - that gives a 36 second buffer which should be plenty.

Reply:
Yeah that makes sense. Can't believe I gave a 120 second buffer 😂